fdbrpc/HTTP.actor.cpp

```diff
@@ -623,7 +688,6 @@ ACTOR Future<Reference<HTTP::IncomingResponse>> doRequestActor(Reference<IConnec
 			if (iid != r->data.headers.end()) {
 				responseID = iid->second;
 			}
-			event.detail("RequestIDReceived", responseID);
```

Comment:
Do we want to add this line back?

Reply:
I removed it as of little use. Added it back.